app/src/main/java/com/example/moviesdatabase/InternetConnectionChecker.java

```diff
@@ -0,0 +1,41 @@
+package com.example.moviesdatabase;
+
+
+import android.content.Context;
+import android.net.ConnectivityManager;
+import android.net.NetworkInfo;
+
+import java.net.InetAddress;
+
+public class InternetConnectionChecker {
+    public boolean isInternetAvailable() {
+        try {
+            InetAddress ipAddr = InetAddress.getByName("google.com");
+            return !ipAddr.equals("");
+
+        } catch (Exception e) {
+            return false;
+        }
+    }
+
+    public static boolean hasConnection(final Context context)
+    {
+        ConnectivityManager cm = (ConnectivityManager)context.getSystemService(Context.CONNECTIVITY_SERVICE);
+        NetworkInfo wifiInfo = cm.getNetworkInfo(ConnectivityManager.TYPE_WIFI);
```

Comment:
`getNetworkInfo` is deprecated, try something like this 
https://stackoverflow.com/questions/32547006/connectivitymanager-getnetworkinfoint-deprecated